src/main/resources/templates/application/partials/opponent-fragments.html

```diff
@@ -0,0 +1,96 @@
+  <div th:replace="~{partials/forms :: dropdown('relationshipToClient', 'Relationship to client', ${relationshipsToClient}, 'code', 'description', false)}"></div>
+  <div th:replace="~{partials/forms :: largeTextInput('otherInformation', 'Any other information about the liable organisation  (Maximum 2000 characters allowed)')}"></div>
+</th:block>
+
+<div th:fragment="createOrganisation"
+     class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <input type="hidden" id="type" name="type" th:value="Organisation"/>
+    <div th:replace="~{partials/forms :: textInput('organisationName', 'Organisation name', 'govuk-input--width-20')}"></div>
+    <div th:replace="~{partials/forms :: dropdownWithDisplayValue('organisationType', 'Organisation type', ${organisationTypes}, 'code', 'description')}"></div>
+    <div th:replace="~{partials/forms :: radios('currentlyTrading', 'Currently trading (optional)', ${currentlyTradingOptions}, 'key', 'value', false)}"></div>
+    <div th:replace="~{partials/forms :: dropdown('relationshipToCase', 'Relationship to case', ${relationshipsToCase}, 'code', 'description', false)}"></div>
+    <div th:replace="~{partials/forms :: dropdown('relationshipToClient', 'Relationship to client', ${relationshipsToClient}, 'code', 'description', false)}"></div>
+    <div th:replace="~{partials/forms :: textInput('contactNameRole', 'Contact name and role (optional)', 'govuk-input--width-20')}"></div>
+    <div th:replace="~{partials/forms :: textInput('houseNameOrNumber', 'Building name / number (optional)', 'govuk-input--width-20')}"></div>
+    <div th:replace="~{partials/forms :: textInput('addressLine1', 'Address line 1 (optional)', 'govuk-input--width-20')}"></div>
+    <div th:replace="~{partials/forms :: textInput('addressLine2', 'Address line 2 (optional)', 'govuk-input--width-20')}"></div>
+    <div th:replace="~{partials/forms :: textInput('city', 'City / town (optional)', 'govuk-input--width-20')}"></div>
+    <div th:replace="~{partials/forms :: textInput('county', 'County (optional)', 'govuk-input--width-20')}"></div>
+    <div th:replace="~{partials/forms :: dropdown('country', 'Country (optional)', ${countries}, 'code', 'description', false)}"></div>
+    <div th:replace="~{partials/forms :: textInput('postcode', 'Post code (optional)', 'govuk-input--width-20')}"></div>
+    <div th:replace="~{partials/forms :: textInput('telephoneWork', 'Telephone (optional)', 'govuk-input--width-20')}"></div>
+    <div th:replace="~{partials/forms :: textInput('faxNumber', 'Fax (optional)', 'govuk-input--width-20')}"></div>
+    <div th:replace="~{partials/forms :: textInput('emailAddress', 'Email address (optional)', 'govuk-input--width-20')}"></div>
+    <div th:replace="~{partials/forms :: largeTextInput('otherInformation', 'Any other information about the liable organisation  (Maximum 2000 characters allowed)')}"></div>
```

Comment:
do we actually limit the form to only allow the 2000 characters?

Reply:
This is handled by the form validator on POST